### PR TITLE
Add interactive list

### PR DESCRIPTION
**Pull request by Addisu87**
I am adding some functionality to my application to make it interactive. I am able to mark task completion by selecting the corresponding checkbox (or undo it by unchecking the checkbox).

### CODE REVIEW

File: modules/removeAll.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  removeAll.classList.add('remove-all');
+  removeAll.innerHTML = 'Remove All';
+
+  clearTasks.appendChild(removeAll);
+
+  removeAll.addEventListener('click', () => {
+    localStorage.clear();
+    DisplayTodos();
+  });
+};
```
Comment (youmari) [inline modules/removeAll.js:18]: - [ ] the `clear all completed ` should remove only the ones that is `completed: true;` your function removes all the todos please fix that and make sure only the completed one are removed 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Addisu87 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary 
✔️ No linter error 
✔️ Github flow 
✔️ Nice Design 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/displayTodos.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      input.checked = todo.done;
+      label.appendChild(input);
+
+      const span = document.createElement('span');
+      span.classList.add('bubble');
+      label.appendChild(span);
+
+      const content = document.createElement('div');
+      content.classList.add('todo-content');
+      content.innerHTML = `<input type="text" value="${todo.content}" readonly>`;
```
Comment (brytebee) [inline modules/displayTodos.js:28]: - [ ] **Optional:** Your todo's display undefined. The `content` attribute should be replaced with `description`.

**Review by brytebee — APPROVED** [review]
Hi, @Addisu87👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**